pycon.py

```diff
@@ -25,6 +25,20 @@
 
 NEWS = get_news()
 
+COVERS = {
+    'cfp': 'img/about/speaker1.jpg',
+    'cfp_form': 'img/about/workshop.jpg',
+    'cfv': 'img/about/cfv.jpg',
+    'coc': 'img/about/chillout.jpg',
+    'countdown': 'img/about/header1.jpg',
+    'index': 'img/about/header1.jpg',
+    'news': 'img/about/news.jpg',
+    'privacy': 'img/about/privacy.jpg',
+    'recording': 'img/about/snake.jpg',
+    'tickets': 'img/about/full_room.jpg',
+    'thanks': 'img/about/header1.jpg',
+}
```

Comment:
I have created a new directory called data,that contain json with data so we can have cleaner file. Could you please create covers.json and move this constant into the json file?

than you can use `read_json_file` method from `utils.py` to get the data easily.

Reply:
If we do my last comment (with css this is irrelevant)